### PR TITLE
Levehnstein change

### PR DESCRIPTION
**Pull request by SimonThordal**
Just a couple of small changes I noticed while looking at other things.

### CODE REVIEW

File: rigour/text/distance.py [hunk 1]
```diff
@@ -55,19 +53,23 @@ def levenshtein_similarity(
         left: A string.
         right: A string.
         max_edits: The maximum number of edits allowed.
-        max_percent: The maximum percentage of edits allowed.
+        max_percent: The maximum fraction of the shortest string that is allowed to be edited.
```
Comment (SimonThordal) [inline rigour/text/distance.py:56]: Maybe it's just me, but I didn't understand this parameter from the comment, so I tried making it a bit clearer. This should be "if max_percent is 0.5 and the shortest word is 4 characters then the max distance is 2 edits".

File: rigour/text/distance.py [hunk 2]
```diff
@@ -55,19 +53,23 @@ def levenshtein_similarity(
...
-    distance = dam_levenshtein(left, right)
     left_len = len(left)
     right_len = len(right)
     if left_len == 0 or right_len == 0:
         return 0.0
+
     # Skip results with an overall distance of more than N characters:
     pct_edits = math.ceil(min(left_len, right_len) * max_percent)
     max_edits_ = min(max_edits, pct_edits) if max_edits is not None else pct_edits
+    if abs(left_len - right_len) > max_edits_:
```
Comment (SimonThordal) [inline rigour/text/distance.py:69]: No need to continue if the difference in length is greater than the number of allowed edits.

File: rigour/text/distance.py [hunk 3]
```diff
@@ -37,8 +37,6 @@ def levenshtein(left: str, right: str) -> int:
     Returns:
         An integer of changed characters.
     """
-    if left == right:
```
Comment (SimonThordal) [inline rigour/text/distance.py:40]: I noticed that Jellyfish makes this check as well.

**Comment by pudo** [comment]
Yep, that makes perfect sense!

**Pull request merged**